src/LoadEarlier.tsx

```diff
@@ -55,68 +56,66 @@ export interface LoadEarlierProps {
   onLoadEarlier?(): void
 }
 
-export default class LoadEarlier extends React.Component<LoadEarlierProps> {
-  static defaultProps = {
-    onLoadEarlier: () => {},
-    isLoadingEarlier: false,
-    label: 'Load earlier messages',
-    containerStyle: {},
-    wrapperStyle: {},
-    textStyle: {},
-    activityIndicatorStyle: {},
-    activityIndicatorColor: 'white',
-    activityIndicatorSize: 'small',
-  }
+export function LoadEarlier(props: LoadEarlierProps): React.ReactElement {
 
-  static propTypes = {
-    onLoadEarlier: PropTypes.func,
-    isLoadingEarlier: PropTypes.bool,
-    label: PropTypes.string,
-    containerStyle: StylePropType,
-    wrapperStyle: StylePropType,
-    textStyle: StylePropType,
-    activityIndicatorStyle: StylePropType,
-    activityIndicatorColor: PropTypes.string,
-    activityIndicatorSize: PropTypes.string,
-  }
-
-  renderLoading() {
-    if (this.props.isLoadingEarlier === false) {
+  const renderLoading = useCallbackOne(() => {
```

Comment:
Please move this code directly into return jsx even if it was into a render function member of the class, as I see there is no render props right?

Reply:
Yeah, I got
Comment, solved